src/components/unique-token/UniqueTokenCard.js

```diff
@@ -1,5 +1,5 @@
 import React, { useCallback, useMemo } from 'react';
-import styled from 'styled-components';
+import styled from 'styled-components/primitives';
```

Comment:
why? Please don't use `styled-components/primitives`. It does not react to our theming mechanisms